modules/ROOT/partials/integrations/angular-tech-ref.adoc

```diff
@@ -83,7 +83,7 @@ Add the `+EditorModule+` to `+@NgModule({imports})+`:
 +
 [source,html]
 ----
-<editor apiKey="test" [init]="{plugins: 'link'}"></editor>
+<editor apiKey="test" [init]="{plugins: 'link'}" />
```

Comment:
```suggestion
<editor apiKey="your-api-key" [init]="{plugins: 'link'}" />
```

Reply:
We need to update to use `no-api-key` for all examples.
```suggestion
<editor apiKey="no-api-key" [init]="{plugins: 'link'}" />
```